src/tests/functional/plugin/shared/include/subgraph_tests/split_concat_memory.hpp

```diff
@@ -28,7 +27,7 @@ TEST_P(SplitConcatMemory, cyclicBufferCorrectness) {
     auto o_tensor = inferRequest.get_tensor(*function->outputs().begin());
     auto output_tensor_ref = ov::Tensor(o_tensor.get_element_type(), o_tensor.get_shape());
 
-    auto fill_by_quarter = [this] (ov::Tensor& tensor, std::vector<float> vals) {
+    auto fill_by_quarter = [this](ov::Tensor& tensor, std::vector<float> vals) {
```

Comment:
Move to `generate_input` map? The structure provide the functors to generate tensors for different ops (https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp)